x/gamm/keeper/pool_service.go

```diff
@@ -57,138 +54,6 @@ func (k Keeper) CalculateSpotPrice(
 	return spotPrice, err
 }
 
-func validateCreatePoolMsg(ctx sdk.Context, msg swaproutertypes.CreatePoolMsg) error {
```

Comment:
Note: moved to swaprouter for now. Incoming https://github.com/osmosis-labs/osmosis/pull/3607 further changes and improves the location of this check